action.yml

```diff
@@ -76,14 +88,22 @@ runs:
     - name: Nodejs - Publish to NPM
       uses: ./.pulumi-package-publish/lang/nodejs
       if: (contains(inputs.sdk, 'nodejs') || (contains(inputs.sdk, 'all') && !contains(inputs.sdk, '!all'))) && !contains(inputs.sdk, '!nodejs')
+      env:
+        NODE_VERSTION: ${{ inputs.node-version }}
```

Comment:
Can we pass this via `with` instead of `env`?

Reply:
I did not find any evidence to this effect - the example given didn't show using `with` on this level: https://docs.github.com/en/actions/creating-actions/creating-a-composite-action

I wouldn't be surprised if it does work but given how long our Actions are taking to kick off today, I'm going to leave this as a refinement. :/